app/styles/ui/_branches.scss

```diff
@@ -4,14 +4,15 @@
   height: 100%;
   display: flex;
   flex-direction: column;
-  width: 365px;
+  min-width: 365px;
+  width: 100%;
 
   & > .tab-bar {
     border-top: var(--base-border);
   }
 
   .branches-list {
-    width: 365px;
+    width: 100%;
```

Comment:
Replace the fixed width and allow the branch list to fill the dropdown container (`.branches-container`).